jms/README.md

```diff
@@ -37,4 +97,12 @@ This object MUST NOT contain any properties. Its name is reserved for future use
 
 ## Message Binding Object
```

Comment:
can you provide some example of a message, the best would be with headers both for the app (`message.headers`) and the protocol headers

Reply:
@derberg Sure! I just appended a commit which provides such an example :-)